docs/sources/alerting/fundamentals/annotation-label/variables-label-annotation.md

```diff
@@ -13,15 +13,11 @@ weight: 117
 
 # How to template annotations and labels
 
-In Grafana it is possible to template annotations and labels just like you would in Prometheus. Those who have used
-Prometheus before should be familiar with the `$labels` variable which holds the label key/value pairs of the alert
-instance and the `$value` variable which holds the evaluated value of the alert instance.
+In Grafana it is possible to template annotations and labels just like you would in Prometheus. Those who have used Prometheus before should be familiar with the `$labels` variable which holds the label key/value pairs of the alert instance and the `$value` variable which holds the evaluated value of the alert instance.
```

Comment:
I removed some newlines where I had written the original documentation in Vim.

Reply:
The extra lines are very helpful when changing text and seeing the changes in GitHub. I recommend leaving them in because it make reviews more efficient.

Reply:
More specifically, break lines into smaller pieces semantically.

Reply:
I want to remove them because small changes (like adding or removing a word) can cause entire lines to be considered "added" or "removed" because it would change the surrounding lines. This will mean in future just the added or removed text is shown in the diff, which I think is what you're asking about?

Reply:
I want to remove them because small changes (like adding or removing a word) can cause entire lines to be considered "added" or "removed" because it would change the surrounding lines. This will mean in future just the added or removed text is shown in the diff, rather than the entire line, which I think is what you're asking about?

Reply:
@jdbaldry, we implemented semantic related lines breaks in Mimir documentation. How do we address @grobinson-grafana’s point about complete lines shifting?

Reply:
Is this just to make it easier when editing docs in Vim and Emacs?

Reply:
Here is an example of what I mean. I added **this is a test comment** to some doc comments in code but its impossible to find amongst all the changed line breaks.

![image](https://user-images.githubusercontent.com/85952834/204262532-225fbab4-c9e2-4f62-b9f1-595933ad904c.png)

Reply:
I've removed all these newline changes so we can make progress on this pull request 🙂